api/resources_portal/views/user.py

```diff
@@ -50,27 +28,11 @@ class UserViewSet(viewsets.ModelViewSet):
     queryset = User.objects.all()
     serializer_class = UserSerializer
 
-    @transaction.atomic
-    def create(self, request, *args, **kwargs):
-        response = super(UserViewSet, self).create(request, args, kwargs)
-
-        # Every user should have their own organization.
-        user = User.objects.get(id=response.data["id"])
-        Organization.objects.create(owner=user)
-
-        return response
-
-    def get_serializer_class(self):
-        if self.action == "create":
-            return CreateUserSerializer
-
-        return UserSerializer
+    http_method_names = ["get", "delete", "put", "patch", "head", "options"]
 
     def get_permissions(self):
         if self.action == "update" or self.action == "partial_update" or self.action == "destroy":
             permission_classes = [IsAuthenticated, IsUserOrAdmin]
-        elif self.action == "create":
-            permission_classes = [AllowAny]
```

Comment:
If POSTs aren't allowed, how are users created?

Reply:
Users are created through the OAuth route. If we want users to be created any other way, we need to get their ORCID, which can only be acquired if you have our client secret. We can still create routes which do this, but just POSTing to user doesn't make very much sense anymore to me. We can discuss further on a call if you like.